src/platform/packages/shared/kbn-discover-contextual-components/src/data_types/logs/components/summary_column/summary_column.tsx

```diff
@@ -65,8 +64,8 @@ const SummaryCell = ({
   const density = maybeNullishDensity ?? DataGridDensity.COMPACT;
   const isCompressed = density === DataGridDensity.COMPACT;
 
-  const rowHeight = maybeNullishRowHeight ?? ROWS_HEIGHT_OPTIONS.single;
-  const isSingleLine = rowHeight === ROWS_HEIGHT_OPTIONS.single || rowHeight === 1;
+  const rowHeight = maybeNullishRowHeight ?? 1;
```

Comment:
What about using `DEFAULT_ROW_COUNT` here?

Reply:
Do you have any existing const in mind or you think about creating one?
If the latter, then we should keep in mind that default row count can be higher then 1 (eg. Discover has 3 by default) - so we would change the logic here.
If you mean just creating a local const for this, then I'll do it right away 😀

Reply:
I was referring to the one you already created in `x-pack/solutions/observability/plugins/logs_explorer/common/constants.ts`

Reply:
Makes perfect sense, will change, thanks!

Reply:
While importing I got `No group crossing error` from eslint. What do you think about creating a local `const` for this module?